src/cgame/cg_servercmds.cpp

```diff
@@ -2377,8 +2380,8 @@ static void CG_ServerCommand(void) {
 
     CG_RemoveChatEscapeChar(text);
     CG_FixLinesEndingWithCaret(text, MAX_SAY_TEXT);
-    s = CG_AddChatModifications(text, Q_atoi(CG_Argv(2)), chatReplay);
-    CG_AddToTeamChat(s, Q_atoi(CG_Argv(2)), chatReplay);
+    s = CG_AddChatModifications(text, clientNum, chatReplay, serverSay);
+    CG_AddToTeamChat(s, clientNum, chatReplay, serverSay);
```

Comment:
Maybe these functions could just take an arbitrary "message type" as an input instead of to two bools? e.g.
```cpp
REPLAY_MSG = 1 << 0;
SERVER_MSG = 1 << 1;

CG_AddToTeamChat(s, clientNum, (REPLAY_MSG | SERVER_MSG))
```